src/conn/conn_oligarch.c

```diff
@@ -79,11 +82,16 @@ __oligarch_metadata_watcher(void *arg)
             }
         }
 
-        buf[last_sep] = '\0';
         name_ptr = last_sep;
         while (name_ptr != 0 && buf[name_ptr - 1] != '\n')
             name_ptr--;
 
+        /* XXX Get the checkpoint metadata from dir_store - just to check that it all works. */
+        WT_ASSERT(session, __wt_disagg_get_meta(session, 0, 0, item) == 0);
```

Comment:
I'm actually going to temporarily disable this test, as it is currently pantry (dir_store) specific, and so that it doesn't create any unexpected fallout when we vendor this back in. Yes, I will use the PALI API to get the checkpoint ID when I get to do this for real!